calico_kubernetes/tests/kube_plugin_test.py

```diff
@@ -16,8 +16,10 @@
 import unittest
 from mock import patch, Mock, call
 from subprocess import CalledProcessError
+import docker
```

Comment:
Don't think you need to import Profile either - you don't need to import things that are being Mocked out.